Test/test_ncSTRING.py

```diff
@@ -0,0 +1,81 @@
+# Automatically adapted for numpy.oldnumeric Aug 01, 2007 by ^F^F^F^F^F
+
+import urllib
+import cdms2
+import MV2
+import os
+import basetest
+import numpy
+
+
+class TestNCString(basetest.CDMSBaseTest):
+    def setUp(self):
+        myurl = "http://uvcdat.llnl.gov/cdat/sample_data/prcp_1951.nc"
+        super(TestNCString, self).setUp()
+        urllib.urlretrieve(myurl, "prcp_1951.nc")
+
+    def tearDown(self):
+        super(TestNCString, self).tearDown()
+        #os.remove("prcp_1951.nc")
+
+
+
+    def testNCStringWrite(self):
+        cdms2.setNetcdf4Flag(1)
+        cdms2.setNetcdfClassicFlag(0)
+        cdms2.setNetcdfShuffleFlag(0)
```

Comment:
shouldn't `cdms2.setNetcdfClassicFlag(0)` be after the next 3 lines? I'm sort of remembering the next 3 turn on netcdf3 classic. But I'm not sure.